osu.Game/Screens/Edit/Compose/Components/BlueprintContainer.cs

```diff
@@ -110,9 +110,9 @@ protected override bool OnMouseDown(MouseDownEvent e)
             bool selectionPerformed = performMouseDownActions(e);
 
             // even if a selection didn't occur, a drag event may still move the selection.
-            prepareSelectionMovement();
+            bool movementPossible = prepareSelectionMovement();
 
-            return selectionPerformed || e.Button == MouseButton.Left;
+            return selectionPerformed || movementPossible;
```

Comment:
The omission of `MouseButton.Left` check broke context menus from displaying, I believe.

Reply:
You're right, I'll investigate.

Reply:
I was side-eyeing this change but let it go for reasons I'm not entirely sure of myself...